.pipelines/.vsts-vhd-automation.yaml

```diff
@@ -8,7 +8,7 @@ steps:
   displayName: 'Image Version Bumping'
 - bash: |
         echo "PR for Release Notes"
-        /bin/bash vhdbuilder/scripts/automate_release_notes.sh $IMAGE_VERSION $BUILD_ID $GITHUB_PAT $ACCESS_TOKEN
+        /bin/bash vhdbuilder/scripts/automate_release_notes.sh $IMAGE_VERSION "${BUILD_ID[@]}" $GITHUB_PAT $ACCESS_TOKEN
```

Comment:
Would this obtain the correct list of build ids for different re-runs?

Reply:
Yes, it would, we only have to supply space separated build IDs. Confirmed via test runs.
https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=48492712&view=results
https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=48492845&view=results
https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=48492580&view=results